toolkit/views/character_generator/character_generator_backend.py

```diff
@@ -317,10 +212,17 @@ def generate_class(self, class_list: list[str], generator_key: str) -> str:
         """
         if generator_key not in self.generators:
             raise RuntimeError("Generator does not exist")
-        for x in class_list:
-            if x not in Character_Generator.CLASS_DICT["All"]:
-                raise RuntimeError("Invalid List")
-        clazz: str = class_list[self.generators[generator_key](0, len(class_list) - 1)]
+        if generator_key not in self.generators:
```

Comment:
in retrospect, it would have been easier to make a get_generator method to deal with the exception. Next time..